core/client/src/test/java/alluxio/client/file/options/CreateDirectoryOptionsTest.java

```diff
@@ -45,14 +52,17 @@ public void fields() {
     Random random = new Random();
     boolean allowExists = random.nextBoolean();
     boolean recursive = random.nextBoolean();
+    Mode mode = new Mode((short) 0123);
```

Comment:
nit: it would be nice to have a createRandom() utility method for the mode class that generates random modes for testing